src/pages/Facility/settings/organizations/FacilityOrganizationIndex.tsx

```diff
@@ -9,38 +10,44 @@ import {
   Card,
   CardContent,
   CardDescription,
-  CardFooter,
   CardHeader,
   CardTitle,
 } from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
 import { Skeleton } from "@/components/ui/skeleton";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
 
 import Page from "@/components/Common/Page";
 import { CardGridSkeleton } from "@/components/Common/SkeletonLoading";
 
 import routes from "@/Utils/request/api";
 import query from "@/Utils/request/query";
-import { FacilityOrganization } from "@/types/facilityOrganization/facilityOrganization";
-
-import CreateFacilityOrganizationSheet from "./components/CreateFacilityOrganizationSheet";
+import CreateFacilityOrganizationSheet from "@/pages/Facility/settings/organizations/components/CreateFacilityOrganizationSheet";
 
 export default function FacilityOrganizationIndex({
   facilityId,
 }: {
   facilityId: string;
 }) {
+  const [expandedRows, setExpandedRows] = useState<Record<string, boolean>>({});
   const { t } = useTranslation();
+  const LIMIT = 1000;
   const { data, isLoading } = useQuery({
     queryKey: ["facilityOrganization", "list", facilityId],
     queryFn: query(routes.facilityOrganization.list, {
       pathParams: { facilityId },
-      queryParams: {
-        parent: "",
-      },
+      queryParams: { limit: LIMIT },
     }),
     enabled: !!facilityId,
   });
-
+  const TableData = data?.results || [];
```

Comment:
```suggestion
  const tableData = data?.results || [];
```

naming convention; capitalization for components.